LocalPackages/SyncUI/Sources/SyncUI/Views/SyncSettingsViewExtension.swift

```diff
@@ -97,6 +107,18 @@ extension SyncSettingsView {
                 })
             })
             .disabled(!model.isAccountRecoveryAvailable)
+            .alert(isPresented: alertBinding) {
```

Comment:
`alertBinding` can be replaced with `$model.shouldShowPasscodeRequiredAlert` here (and save you the hassle of defining the binding).

```suggestion
            .alert(isPresented: $model.shouldShowPasscodeRequiredAlert) {
```